apps/rush-lib/src/scripts/install-run-rushx.ts

```diff
@@ -0,0 +1 @@
+import "./install-run-rush.ts";
```

Comment:
I think this should just be
```suggestion
import './install-run-rush';
```